### PR TITLE
Recommend forcing npx to use the latest version of the migration script

### PR DESCRIPTION
**Pull request by ivomurrell**
# Description

I always include the `@latest` specifier when people ask the easiest way to migrate to Tool Kit these days, but I never got round to updating the documentation so that this wasn't hidden knowledge. Finally pushed by @apaleslimghost posting about this paper cut!

# Checklist:

- [x] My branch has been rebased onto the latest commit on main (don't merge main into your branch)
- [x] My commit messages are [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), for example: `feat(circleci): add support for nightly workflows`, `fix: set Heroku app name for staging apps too`

### CODE REVIEW

File: docs/migrating-to-tool-kit.md [hunk 1]
```diff
@@ -28,13 +28,15 @@ to follow along with.
 The migration tool is published to npm as `@dotcom-tool-kit/create`. Thanks to
 some [syntax sugar
 magic](https://docs.npmjs.com/cli/v7/commands/npm-init#description) provided by
-npm's `npm init` script this means you can just run
```
Comment (apaleslimghost) [inline docs/migrating-to-tool-kit.md:31]: **praise**: yessss

File: scripts/create-plugin.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ fs.mkdirSync(directory)
 process.chdir(directory)
 
 console.log('📦 initialising package')
-execSync('npm init -y --scope @dotcom-tool-kit')
+execSync('npm init -y --scope @dotcom-tool-kit@init')
```
Comment (apaleslimghost) [inline scripts/create-plugin.js:17]: **chore**: this is unrelated (and will probably break!), this call is just for creating the `package.json` in a new plugin

**thought**: do we still use this script? does it even still work? maybe let's just delete it?

**Review by apaleslimghost — APPROVED** [review]

File: scripts/create-plugin.js (hunk 2 above)
Comment (ivomurrell) [reply scripts/create-plugin.js:17]: oops yeah, thrown off by the scope argument! I feel like we only don't use this script because we aren't in the habit of creating new plugins 😅 but I think we should keep it, if not just to have a bit of documentation on all the things you need to look out for when creating a plugin.
